### DISCUSSION TITLE
Vicuna 13B 4Bit Unfiltered

### DISCUSSION THREAD

**Discussion by InfernalDread** [keldenl/gpt-llama.cpp#8] [Q&A] [answered]
Does anyone have the huggingface link to download the ggml for the model in the title?

**Comment by keldenl** [comment, marked as answer]
please don't link llama or any of the variant models in this repo -- just search the keywords in hf and u should be able to find it

**Reply by InfernalDread** [reply]
oh, alrighty, never knew it was a nono. my bad
